src/de/duenndns/ssl/MemorizingTrustManager.java

```diff
@@ -495,11 +495,9 @@ private static String certHash(final X509Certificate cert, String digest) {
 		}
 	}
 
-	private static void certDetails(StringBuilder si, X509Certificate c) {
```

Comment:
is there any reason to un-static that function?

Reply:
I would like to have access to `master.getString` inside it.

Reply:
Oh dear, that whole function needs to be i18ned.